tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraCalendarEventAcceptMethodContract.scala

```diff
@@ -1039,44 +1057,100 @@ trait LinagoraCalendarEventAcceptMethodContract {
     }
   }
 
+  @Test
+  def shouldFailWhenBlobIdIsNotPrefixedByMessageId(): Unit = {
```

Comment:
Note: This test is only appropriate for `StandaloneEventAttendanceRepository`, as it is the only implementation currently available. Once a CarDav repository is implemented, this test should be moved to a standalone-specific location.